extensions/sips/package.json

```diff
@@ -48,15 +31,7 @@
       "name": "rotate",
       "title": "Rotate Images",
       "description": "Rotate the selected images by the specified amount of degrees",
-      "mode": "no-view",
-      "arguments": [
-        {
-          "name": "degrees",
-          "placeholder": "Degrees",
-          "type": "text",
-          "required": true
-        }
-      ]
```

Comment:
I think the arguments were removed by mistake? They're needed for the command to be useful.